integration/hurl/tests_ok/help.out.pattern

```diff
@@ -125,4 +125,4 @@ Options:
   -h, --help
           Print help
   -V, --version
-          Print version
+          Print version
```

Comment:
The new line musn't be deleted (that's why some integration test are KO)

Reply:
I think the new line is needed... Since this tests are failing